### ISSUE TITLE
Consider improving documentation style

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/center-and-variability#490]
For code review #447 ...

I made this same comment recently for Kepler's Laws code review.  I'm fine if you do nothing and close this issue. But please consider this feedback for future documentation that you write at PhET and beyond.

PhET's code documentation style is not really up to industry standards.  A lot of it (perhaps the majority?) is a terse shorthand, not full sentences, not grammatically correct, dodgy punctuation, mixed tenses, typos, etc.  (And yes, there are plenty of examples of where I've been guilty of this too.)

Two things about this concern me:
(1) Newer developers are adopting this style, to their professional detriment.
(2) It's not a good look for public-facing code, something that is more important with POSE and PhET-iO.

So I'd like to see PhET developers put a little more "polish" on their documentation. 

For example (and not to pick on anyone), here's the documentation for DragIndicatorModel. It consists of sentence fragments, dodgy grammar, mixed tense,...

```ts
/**
 * A model that tracks whether soccerBalls have been dragged or not.
 * Will also determine if the dragIndicator needs to be visible.
```

Compare to:

```ts
/**
 * DragIndicatorModel tracks whether a soccer ball has been dragged, and whether the 
 * drag indicator needs to be visible.
```

**Comment by samreid** [comment]
A lot of this was also covered in #479.

**Comment by samreid** [comment]
I made more improvements for soccer-common. Let's touch base and decide if/how much to do this for center-and-variability too.

**Comment by matthew-blackman** [comment]
@samreid @marlitas agree that this is a good idea stylistically, and would like to address this issue while the sim is undergoing dev testing. I will self-assign and prioritize accordingly.

**Comment by marlitas** [comment]
Wow. So much great work here @matthew-blackman. I skimmed through all of it and it is such a HUGE improvement. Thanks so much. There was one little thing that I fixed in the median screen, but other than that everything looks great. I personally think this is good to close. If anyone else would like to re-open please feel free to do so.

**Comment by pixelzoom** [comment]
Reopening, because it sounds like @matthew-blackman wanted @pixelzoom, @samreid, and @marlitas to have a look.

**Comment by pixelzoom** [comment]
Reopening, because based on our discussion in standup, @matthew-blackman wanted me to have a look.

**Comment by pixelzoom** [comment]
As @marlitas said... great work here.  I did not ready ever soure file, but made a fairly large sample. The documentation is much more useful, and easier to read.  Thanks for doing this work.

I made one general/minor change in above commit, which is to differentiate between property and Property - they are different things.

It would be nice to imporve the doc similarly for soccer-common.  It looks there has been some work there, but there is still a fair amount of old-style doc.  I'll let the CaV team decide whether to do that as part of this issue, create a future issue in soccer-common, ignore it, or whatever. 

Back to @matthew-blackman.
